### PR TITLE
feat(Google Meet): add settings and use a cache

### LINKED ISSUE #8608: Google Meet | meet.google.com - add title of the room in presence display
### Presence name

Google Meet

### Description

Can we not have title of the room in the response of presence status? Just like we get the number of people in the room?

### Alternatives

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Resolves #8608
## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

**Comment by DarkVillager** [comment]
_This made me rage 3-4 times_

File: websites/G/Google Meet/presence.ts [hunk 1]
```diff
@@ -1,25 +1,167 @@
...
+		} else {
+			pdCache = {
+				name: "",
+				details: presenceData?.details,
+				state: presenceData?.state,
+				meetTitle: presenceName ?? "",
+				meetDetails: presenceDetail ?? "",
+				meetState: presenceState ?? "",
+			};
+		}
```
Comment (Bas950) [inline websites/G/Google Meet/presence.ts:160]:
```suggestion
		}
		pdCache = {
			name: presenceData?.name ?? "",
			details: presenceData?.details,
			state: presenceData?.state,
			meetTitle: presenceName ?? "",
			meetDetails: presenceDetail ?? "",
			meetState: presenceState ?? "",
		};
```

File: websites/G/Google Meet/presence.ts [hunk 2]
```diff
@@ -1,25 +1,167 @@
...
+		document.querySelector('[id="ucc-8"]')?.textContent ??
+		document.querySelector('[jsname="r4nke"]')?.textContent;
+
+	const meetTime = document.querySelector('[jsname="W5i7Bf"]')?.textContent,
+		meetPeople = document.querySelector(".uGOf1d")?.textContent;
+
+	if (
+		(!meetTitle ||
+			meetTitle?.includes("Read to join") ||
+			meetTitle?.includes("left the meeting") ||
```
Comment (Bas950) [inline websites/G/Google Meet/presence.ts:96]: Just wondering, if you browser language is in another language than english doesn't google meet show different text? So not sure if this includes will succeed in those languages.

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/G/Google Meet/presence.ts (hunk 2 above)
Comment (DarkVillager) [reply websites/G/Google Meet/presence.ts:96]: I forgot to check woops

**Review by Bas950 — APPROVED** [review]

**Review by theusaf — APPROVED** [review]

**Pull request merged**